src/app/clusters/ota-requestor/OTARequestor.h

```diff
@@ -75,7 +74,13 @@ class OTARequestor : public OTARequestorInterface
     // Virtual functions from OTARequestorInterface -- end
 
     /**
-     * Called to establish a session to mProviderNodeId on mProviderFabricIndex
+     * Set the server instance
```

Comment:
The name makes it obvious. The comment should express something like:

```suggestion
     * Set the server instance used to get access to the system resources necessary to
     * open CASE sessions and drive BDX transfers.
```